business/checkers/sidecars/egress_listener_checker.go

```diff
@@ -92,9 +91,6 @@ func (elc EgressHostChecker) HasMatchingService(host kubernetes.Host, itemNamesp
 	if host.IsWildcard() && host.Namespace == itemNamespace {
```

Comment:
This validation also shows some regression as well (not sure if introduced by this PR or during all this refactoring).
![image](https://user-images.githubusercontent.com/1662329/151190521-d3d957fa-06e2-4bdc-b82b-3f3dd3bdbab0.png)